tests/test_expertise_service.py

```diff
@@ -472,7 +472,7 @@ def test_request_journal(self, openreview_context, celery_session_app, celery_se
                     },
                     "entityB": { 
                         'type': "Note",
-                        'id': 'KHnr1r7H'
+                        'id': '[KHnr1r7H]'
```

Comment:
I'm not sure why this is an array now.

Reply:
I think I did it by mistake, that's why my tests are failing. Will fix this.